free-tunes/Controllers/OnBoardingViewController.swift

```diff
@@ -9,7 +9,10 @@ import UIKit
 
 class OnBoardingViewController: UIViewController {
 
-    @IBOutlet weak var headerText: UILabel!
+// MARK: - Interface Builder Outlets
+    @IBOutlet private weak var headerText: UILabel!
```

Comment:
As a suggestion I would Make every IBOutlet and IBAction private

Reply:
I believe it is the case unless if i'm missing something